src/CoreBundle/Resources/views/snippets/common/download/download.html.twig

```diff
@@ -0,0 +1,22 @@
+{# @var downloadLinkDataModel \ChameleonSystem\CoreBundle\DataModel\DownloadLink #}
+
+{% if downloadLinkDataModel.backendLink %}
```

Comment:
The variable/method is named `isBackendLink`?

Reply:
yes. DownloadLink::isBackendLink